media-libs/osl/osl-1.10.5.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit cmake-utils toolchain-funcs
+
+DESCRIPTION="Advanced shading language for production GI renderers"
+HOMEPAGE="http://opensource.imageworks.com/?p=osl"
+SRC_URI="https://github.com/imageworks/OpenShadingLanguage/archive/Release-${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+X86_CPU_FEATURES=(
+	sse2:sse2 sse3:sse3 ssse3:ssse3 sse4_1:sse4.1 sse4_2:sse4.2
+	avx:avx avx2:avx2 avx512f:avx512f f16c:f16c
+)
+CPU_FEATURES=( ${X86_CPU_FEATURES[@]/#/cpu_flags_x86_} )
+
+IUSE="doc partio qt5 test ${CPU_FEATURES[@]%:*}"
+
+RDEPEND="
+	>=dev-libs/boost-1.62:=
+	dev-libs/pugixml
+	>=media-libs/openexr-2.2.0:=
+	>=media-libs/openimageio-1.8.5
+	sys-devel/clang:7=
```

Comment:
Are you sure about that though?

Reply:
It needs >=clang-3.4 but >=llvm:5, if llvm is installed beside clang. I was not sure, how best to handle this without inheriting llvm.eclass.

Reply:
According to https://github.com/imageworks/OpenShadingLanguage/blob/master/INSTALL.md it should work with llvm 5, 6, 7 and 8.

Reply:
Why would it not work with slot 8, e.g.?

Reply:
It should work with slot 8 too. So I guess, I need to use the llvm eclass for this.